.gitignore

```diff
@@ -8,3 +8,4 @@ compile_commands.json
 cmake-build-*
 Testing/
 configs/static_config.yaml
+.vscode/
```

Comment:
For consistency, please move it right below `.idea/`

Reply:
Done